howto/instance/create.md

```diff
@@ -71,6 +71,22 @@ Launch a raw instance by providing the image ID in the following command:
 
 See [Provided images](https://discourse.ubuntu.com/t/provided-images/24185) for a list of images that are available in Anbox Cloud.
 
+### Launch an instance with streaming enabled
+
+*since 1.23.0*
```

Comment:
The more we all start to contribute doc updates in advance, the more I feel like we should have separate branches for each release version. But that's a separate conversation. Let's discuss it further.